src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,48 +1,146 @@
-  public RobotContainer() {
-    // Configure the button bindings
-    configureButtonBindings();
-  }
-
-  /**
-   * Use this method to define your button->command mappings. Buttons can be created by
-   * instantiating a {@link GenericHID} or one of its subclasses ({@link
-   * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
-   * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
-   */
-  private void configureButtonBindings() {}
-
-  /**
-   * Use this to pass the autonomous command to the main {@link Robot} class.
-   *
-   * @return the command to run in autonomous
-   */
-  public Command getAutonomousCommand() {
-    // An ExampleCommand will run in autonomous
-    return m_autoCommand;
-  }
+	// The robot's subsystems and commands are defined here...
+
+	public final DriveSubsystem driveSubsystem = new DriveSubsystem();
+	// public final ServoSubsystem servoSubsystem = new ServoSubsystem();
+
+	/**
+	 * The container for the robot. Contains subsystems, OI devices, and commands.
+	 * 
+	 */
+
+	// Tank Drive
+	private XboxController driver;
+	private Joystick leftJoystick;
+	private Joystick rightJoystick;
+	private JoystickButton buttonA;
+	private JoystickButton buttonB;
+	private JoystickButton buttonX;
+	private JoystickButton buttonY;
+
+	// Arcade Drive
+	private Joystick wheel;
+	private Joystick throttle;
+	private Joystick buttonPanel;
+	public RobotContainer() {
+		// Configure the button bindings
+		configureButtonBindings();
+	}
+
+	/**
+	 * Use this method to define your button->command mappings. Buttons can be
+	 * created by instantiating a {@link GenericHID} or one of its subclasses
+	 * ({@link edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then
+	 * passing it to a {@link edu.wpi.first.wpilibj2.command.button.JoystickButton}.
+	 */
+	private void configureButtonBindings() {
+		driver = new XboxController(Constants.XBOX_CONTROLLER_PORT);
+		// Tank Drive
+		leftJoystick = new Joystick(Constants.LEFT_JOYSTICK_PORT);
+		rightJoystick = new Joystick(Constants.RIGHT_JOYSTICK_PORT);
+		buttonA = new JoystickButton(driver, Constants.BUTTON_A);
+		buttonB = new JoystickButton(driver, Constants.BUTTON_B);
+		buttonX = new JoystickButton(driver, Constants.BUTTON_X);
+		buttonY = new JoystickButton(driver, Constants.BUTTON_Y);
+
+		// Arcade Drive
+		wheel = new Joystick(Constants.WHEEL_PORT);
+		throttle = new Joystick(Constants.THROTTLE_PORT);
+		buttonPanel = new Joystick(Constants.BUTTON_PANEL_PORT);
+	}
+
+	/**
+	 * Use this to pass the autonomous command to the main {@link Robot} class.
+	 *
+	 * @return the command to run in autonomous
+	 */
+	public Command getAutonomousCommand() {
+		// An ExampleCommand will run in autonomous
+
+		return null; // no auto command yet
+	}
+
+	public XboxController getDriver() {
+		return driver;
+	}
+
+	public DriveSubsystem getDriveSystem() {
+		return driveSubsystem;
+	}
+
+	public Joystick getLeftJoystick() {
```

Comment:
All of these get* functions are unnecessary right? The only object that should be handing subsystems/inputs is the RobotContainer, which can already access these members directly.